src/SignalR/clients/java/signalr/.gitignore

```diff
@@ -2,12 +2,13 @@
 .gradletasknamecache
 .gradle/
 build/
-/test-results
+test/test-results
 .settings/
 out/
 *.class
 *.jar
 .project
 .classpath
+core/src/main/java/com/microsoft/signalr/Version.java
```

Comment:
Nit: Should we just add a .gitignore to `core/src/main/java/com/microsoft/signalr/`? It would make things easier if we ever change the directory layout in the future.

Reply:
Seems like a good idea, but I'd prefer to do it as followup - CI is flaky right now due to https://github.com/dotnet/aspnetcore/issues/25293, so once it goes green here I'd prefer to stop churning. I'll follow issues to do this, figure out the VSCode issue, and fix the docs as mentioned in the PR description

Reply:
Had to iterate on this anyways (skipping helix for now to get this in for rc1, will fix for rc2), so I fixed the .gitignore: https://github.com/dotnet/aspnetcore/pull/25253/commits/7a3ca89c7b00d104eb3f969bfc4e59bc747de2d1